pkg/api/types.go

```diff
@@ -1303,6 +1304,14 @@ type PodSpot struct {
 	// Notify string     `json:"notify"`
 }
 
+// PodDedicated is the policy that determines if a dedicated host instance may
+// be used for a pod this is currently unimplemented but could be useful for
+// future use in dedicated host support it can more or less mimic the way that
+// PodSpot works with a custom type and "enum" like characteristics
+type PodDedicated struct {
+	// TODO
+}
+
```

Comment:
Do you think we'll use this in the near future? If not, it might be better to remove it from this change.

Reply:
I have removed this, we can add this at a later date if we deem it necessary.